### PR TITLE
PostgreSQL 13 support

### PR DESCRIPTION
**Pull request by erma07**
Modified the code to support PostgreSQL v13

### CODE REVIEW

**Comment by df7cb** [comment]
Hi,
thanks for working on this, Debian needs a new cstore_fdw package to proceed with migrating to PG13.

Unfortunately, there's a problem in the regression tests:
```
2020-12-04 11:37:25.266 CET [326277] LOG:  server process (PID 326329) was terminated by signal 11: Segmentation fault
2020-12-04 11:37:25.266 CET [326277] DETAIL:  Failed process was running: COPY contestant FROM '/srv/projects/postgresql/cstore-fdw/cstore-fdw/data/contestants.1.csv' WITH CSV;
2020-12-04 11:37:25.266 CET [326277] LOG:  terminating any other active server processes
```

**Comment by erma07** [comment]
My bad, had no way to test pre 13 versions and regression tests here takes ages. tuptoaster.h for  <= 12 was missing. Should be working now.

File: Makefile [hunk 1]
```diff
@@ -46,7 +46,7 @@ ifndef MAJORVERSION
     MAJORVERSION := $(basename $(VERSION))
 endif
 
-ifeq (,$(findstring $(MAJORVERSION), 9.3 9.4 9.5 9.6 10 11 12))
+ifeq (,$(findstring $(MAJORVERSION), 9.3 9.4 9.5 9.6 10 11 12 13))
     $(error PostgreSQL 9.3 to 12 is required to compile this extension)
```
Comment (onlined) [inline Makefile:50]:
```suggestion
    $(error PostgreSQL 9.3 to 13 is required to compile this extension)
```

**Review by onlined — CHANGES_REQUESTED** [review]

**Comment by StepanYankevych** [comment]
Guys, any estimates when PG13 support can be released? 
We plan to migrate to PG13 some on env and would be great to use cstore in PG13.

**Comment by pykello** [comment]
@StepanYankevych pg13 was merged in https://github.com/citusdata/cstore_fdw/pull/243. I will close this PR.

**Pull request closed without merging**